bgpd/bgp_evpn_vty.c

```diff
@@ -1887,6 +1887,13 @@ static struct bgpevpn *evpn_create_update_vni(struct bgp *bgp, vni_t vni)
 
 	vpn = bgp_evpn_lookup_vni(bgp, vni);
 	if (!vpn) {
+		/* Check if this L2VNI is already configured as L3VNI */
```

Comment:
In addition to the log message, will there be any error shown to user on CLI  when this condition happens ?

Reply:
Yes, the user would be notified with generic message failed to create vni and command would be rejected. This is an existing behavior for other reasons if vni creation failed.